### PR TITLE
[test] add some auto test cases.

### PR DESCRIPTION
**Pull request by liikeuforever**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [ ] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?
572 652 669 784 799 822 830 842 877 899 923 994 1076 1148 1867
https://github.com/cnosdb/cnosdb/issues/572
https://github.com/cnosdb/cnosdb/issues/652
https://github.com/cnosdb/cnosdb/issues/669
https://github.com/cnosdb/cnosdb/issues/784
https://github.com/cnosdb/cnosdb/issues/799
https://github.com/cnosdb/cnosdb/issues/822
https://github.com/cnosdb/cnosdb/issues/830
https://github.com/cnosdb/cnosdb/issues/842
https://github.com/cnosdb/cnosdb/issues/877
https://github.com/cnosdb/cnosdb/issues/899
https://github.com/cnosdb/cnosdb/issues/923
https://github.com/cnosdb/cnosdb/issues/994
https://github.com/cnosdb/cnosdb/issues/1076
https://github.com/cnosdb/cnosdb/issues/1148
https://github.com/cnosdb/cnosdb/issues/1867

Related #.

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

File: e2e_test/src/independent/client_tests.rs [hunk 1]
```diff
@@ -362,3 +358,116 @@ fn client_start_test() {
...
+
+    assert_eq!(
+        resp.text().unwrap(),
+        "day,day_temperature,station,day,day_speed,station\n2022-02-06T00:00:00.000000000,66.94915254237289,LianYunGang,2022-02-06T00:00:00.000000000,65.55,XiaoMaiDao\n"
+    );
+}
+
+#[test]
+#[serial]
+#[ignore]
```
Comment (Subsegment) [inline e2e_test/src/independent/client_tests.rs:436]: why ignore

Comment (liikeuforever) [reply e2e_test/src/independent/client_tests.rs:436]: this test can pass in my env.
MakeFile CI run this test: input_partitions=6, split_num=6
Local run this test: input_partitions=1, split_num=1
so i ignore it to pass the CI.
If this approach is not quite appropriate, I will make the necessary modifications.

Comment (Subsegment) [reply e2e_test/src/independent/client_tests.rs:436]: These configurations are also configurable locally

Comment (liikeuforever) [reply e2e_test/src/independent/client_tests.rs]: the parameters that are not significantly relevant to the issue or bug have been handled.

**Review by Subsegment — APPROVED** [review]

**Pull request merged**